chapter2/TodoListAPI/TodoListAPI.csproj

```diff
@@ -15,7 +15,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Identity.Web" Version="0.3.0-preview" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="0.4.0-preview" />
+    <PackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" Version="0.4.0-preview" />
```

Comment:
Why do we need Microsoft.Identity.Web.MicrosoftGraph? 
the Startup.cs does not use .AddMicrosoftGraph

(it should, BTW)

Reply:
@jmprieur just fyi, @Shama-K 's PR is using .addMicrosoftGraph (for the overage scenario)